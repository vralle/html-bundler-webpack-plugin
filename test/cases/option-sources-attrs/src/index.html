<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
  </head>
  <body>
    <h1>Home</h1>

    <!-- data-src and data-srcset attributes -->
    <img data-src="@images/apple.png" data-srcset="@images/fig.png, @images/fig2.png 100w, @images/fig3.png 1.5x">

    <img
      data-srcset="@images/fig.png, @images/fig2.png 100w, @images/fig3.png 1.5x"
      srcset="@images/lemon.png, @images/lemon2.png 200w, @images/lemon3.png 2.5x"
      src="@images/apple.png"
    >

    <!-- custom attributes -->
    <img
      data-ignore-src="@images/fig.png"
      data-resolve-src="@images/fig2.png"
      alt="resolve custom attribute"
    />

    <!-- new custom tag with new attributes -->
    <img-ng
      data-src-one="@images/fig.png"
      data-src-two="@images/fig2.png"
      alt="custom image"
    ></img-ng>

    <svg width="160" height="130">
      <image href="@images/fig.png"></image>
    </svg>

    <svg width="24" height="24">
      <use href="@images/icons/shapes.svg#circle"></use>
    </svg>

    <svg width="24" height="24">
      <use href="@images/icons/shapes.svg#rect"></use>
    </svg>

  </body>
</html>