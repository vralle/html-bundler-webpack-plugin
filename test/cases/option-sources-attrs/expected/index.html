<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
  </head>
  <body>
    <h1>Home</h1>

    <!-- data-src and data-srcset attributes -->
    <img data-src="assets/img/apple.02a7c382.png" data-srcset="assets/img/fig.c6809878.png, assets/img/fig2.c6809878.png 100w, assets/img/fig3.c6809878.png 1.5x">

    <img
      data-srcset="assets/img/fig.c6809878.png, assets/img/fig2.c6809878.png 100w, assets/img/fig3.c6809878.png 1.5x"
      srcset="assets/img/lemon.7b66be8e.png, assets/img/lemon2.7b66be8e.png 200w, assets/img/lemon3.7b66be8e.png 2.5x"
      src="assets/img/apple.02a7c382.png"
    >

    <!-- custom attributes -->
    <img
      data-ignore-src="@images/fig.png"
      data-resolve-src="assets/img/fig2.c6809878.png"
      alt="resolve custom attribute"
    />

    <!-- new custom tag with new attributes -->
    <img-ng
      data-src-one="assets/img/fig.c6809878.png"
      data-src-two="assets/img/fig2.c6809878.png"
      alt="custom image"
    ></img-ng>

    <svg width="160" height="130">
      <image href="assets/img/fig.c6809878.png"></image>
    </svg>

    <svg width="24" height="24">
      <use href="assets/img/shapes.b9fc57c6.svg#circle"></use>
    </svg>

    <svg width="24" height="24">
      <use href="assets/img/shapes.b9fc57c6.svg#rect"></use>
    </svg>

  </body>
</html>